taichi/python/export_ggui.cpp

```diff
@@ -232,13 +239,16 @@ struct PyScene {
     renderable_info.draw_first_index = (int)draw_first_index;
     renderable_info.draw_vertex_count = (int)draw_vertex_count;
     renderable_info.draw_first_vertex = (int)draw_first_vertex;
+    renderable_info.display_mode = display_mode;
 
     MeshInfo info;
     info.renderable_info = renderable_info;
     info.color = tuple_to_vec3(color);
     info.two_sided = two_sided;
     if (transforms.valid) {
-      info.num_instances = transforms.shape[0];
+      info.num_instances = draw_instance_count > transforms.shape[0]
```

Comment:
Maybe  `draw_instance_count > transforms.shape[0] - instance_offset` ? or something else.

Reply:
Thanks, have solved this problem.